openstack/networking/v2/extensions/attributestag/results.go

```diff
@@ -0,0 +1,26 @@
+package attributestag
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// "tagResult" denoting that "tags" are returned and can be used for both the ReplaceAll and Get actions
+
```

Comment:
nit: let's remove the empty line. This comment was also just for the example I gave. You can either remove it or do something more formal like:

```
// tagResult represents a result which will contain a resource's tags.
```